Mooreds/add diagram to core concepts pages (#2511)
FusionAuth/fusionauth-site: site/docs/v1/tech/guides/user-actions.adoc

* Set the `CLIENT_ID` and `CLIENT_SECRET` in your `.env` file to the values you recorded for the new PiedPiper Application in this link:#create-piedpiper-application[section].
* Create a new environment variable in the `.env` file called `API_KEY` and set the value to the value of the API key you created earlier `FTQkSoanK7ObbNjOoU69WDVclfTx8L_zfEJbdR8M0xu-jKotV0iQZiQh`.
* Note in the `package.json` file that the `@fusionauth/typescript-client` library is available for use. This is what will be calling the FusionAuth API to create Action instances.

=== Create Mock Intercom API

In the `fusionauth-example-5-minute-guide` Node.js app, open `app.js`.

You'll add a new route that pretends to be Intercom and will listen for new subscribers to start the onboarding process. In this tutorial, the API will just print the webhook to the console so that you can see what it looks like.

> At the very top of the file, add a reference to the FusionAuth TypeScript client library, the `API_KEY` and `BASE_URL` environment variables.

Reviewer: I had a problem here.  I added to the very top of the file, but when the application ran the variables were undefined.  I found I had to move the declaration of the variables after the var dotenv = require('dotenv').config();
```suggestion
After the line `var indexRouter = require('./routes/index');`, add a reference to the FusionAuth TypeScript client library, the `API_KEY` and `BASE_URL` environment variables.
```
